sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -389,6 +389,50 @@ case class TruncateTableCommand(
 }
 
 /**
+ * A command to repair a table by discovery all the partitions in the directory.
+ *
+ * The syntax of this command is:
+ * {{{
+ *   MSCK REPAIR TABLE table_name;
+ * }}}
+ *
+ * This command is the same as AlterTableRecoverPartitions
+ */
+case class RepairTableCommand(tableName: TableIdentifier) extends RunnableCommand {
+  override def run(spark: SparkSession): Seq[Row] = {
+    val catalog = spark.sessionState.catalog
+    val table = catalog.getTableMetadata(tableName)
+    if (!catalog.tableExists(tableName)) {
```

Comment:
This is a dead code. The previous line already checks whether the table exists or not.
```
val table = catalog.getTableMetadata(tableName)
```